scripts/packages/chocolatey/bazel.nuspec.template

```diff
@@ -75,8 +75,9 @@ Supply like `--params="/option:value ..."` ([see docs for --params](https://gith
       <dependency id="chocolatey-uninstall.extension" />
     </dependencies>-->
     <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.0.7"/>
       <dependency id="jdk8" version="[8.0.102,)"/>
-      <dependency id="msys2" version="[20160719.1,)"/>
+      <dependency id="msys2" version="[20160719.1.0]"/>
```

Comment:
You asked me to pin it to the precise version inside https://github.com/bazelbuild/bazel/issues/2449#issuecomment-278059161 on the basis that others didn't work for bazel (at the time). I haven't tested that assertion, and also haven't seen issues reported about bazel failures against later versions (there's `20160719.1.1`, and there are also later versions in the [moderation queue (this is a paged link)](https://chocolatey.org/packages?sortOrder=package-title&page=7&prerelease=False&moderatorQueue=True)).